src/components/FlatListItem/index.js

```diff
@@ -44,38 +41,38 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
     paddingVertical: 16,
     borderBottomWidth: 1,
-    borderBottomColor: colors.alto,
+    borderBottomColor: COLORS.alto,
     paddingHorizontal: 20,
-    alignItems: 'center',
+    alignItems: 'center'
   },
   label: {
     fontFamily: fonts.inter[400],
     fontSize: 14,
     fontWeight: '400',
-    color: colors.black,
+    color: COLORS.black
   },
   desc: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
     fontWeight: '400',
-    color: colors.gray,
-    flexWrap: 'wrap',
+    color: COLORS.gray8,
+    flexWrap: 'wrap'
   },
   labelIcon: {
     fontFamily: fonts.inter[500],
     fontSize: 14,
     fontWeight: '500',
-    color: colors.black,
+    color: COLORS.black
   },
   contentText: {
     flexDirection: 'row',
     alignItems: 'center',
     flexWrap: 'nowrap',
-    width: 'auto',
+    width: 'auto'
   },
   containerLabel: {
     paddingLeft: 12,
     flexWrap: 'nowrap',
-    width: '88%',
-  },
+    width: '88%'
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been updated from `colors` to `COLORS`. Ensure that these changes are consistent across all components and files to maintain uniformity.


<!-- This is an auto-generated comment by OSS CodeRabbit -->